src/main/java/mil/dds/anet/resources/ReportResource.java

```diff
@@ -747,12 +774,57 @@ public Response showRollupEmail(@Auth Person user, @QueryParam("startDate") Long
 			return Utils.resultGrouper(list, "stats", "personId", tlf);
 		}
 	}
-
+	
 	private Map<Integer, Organization> getOrgsByShortNames(List<String> orgShortNames) {
-		final Map<Integer, Organization> result = new HashMap<>();
-		for (final Organization organization : engine.getOrganizationDao().getOrgsByShortNames(orgShortNames)) {
-			result.put(organization.getId(), organization);
+			final Map<Integer, Organization> result = new HashMap<>();
+			for (final Organization organization : engine.getOrganizationDao().getOrgsByShortNames(orgShortNames)) {
+				result.put(organization.getId(), organization);
+			}
+			return result;
+		}
+	/**
+	 * The comparator to be used when ordering the roll up graph results to ensure
+	 * that any pinned organisation names are returned at the start of the list.
+	 * 
+	 * @author timothy.ward
+	 *
+	 */
+	public static class RollupGraphComparator implements Comparator<RollupGraph> {
+
+		private final List<String> pinnedOrgNames;
+
+		/**
+		 * Creates an instance of this comparator using the supplied pinned organisation
+		 * names.
+		 * 
+		 * @param pinnedOrgNames
+		 *            the pinned organisation names
+		 */
+		public RollupGraphComparator(final List<String> pinnedOrgNames) {
+			this.pinnedOrgNames = pinnedOrgNames;
+		}
+
+		/*
+		 * (non-Javadoc)
+		 * 
+		 * @see java.util.Comparator#compare(java.lang.Object, java.lang.Object)
+		 */
+		@Override
+		public int compare(final RollupGraph o1, final RollupGraph o2) {
+
+			if (pinnedOrgNames.contains(o1.getOrg().getShortName())) {
+				if (pinnedOrgNames.contains(o2.getOrg().getShortName())) {
+					return pinnedOrgNames.indexOf(o1.getOrg().getShortName())
+							- pinnedOrgNames.indexOf(o2.getOrg().getShortName());
+				} else {
+					return -1;
+				}
+			} else if (pinnedOrgNames.contains(o2.getOrg().getShortName())) {
+				return 1;
+			} else {
+				return o1.getOrg().getShortName().compareTo(o2.getOrg().getShortName());
```

Comment:
Since multiple orgs can have the same shortName, it would probably be better to use something like:
```
final int c = org1.getShortName().compareTo(org2.getShortName());
if (c != 0) {
  return c;
}
return org1.getId() - org2.getId();
```